oci/archive/oci_src.go

```diff
@@ -28,6 +28,18 @@ func (e ImageNotFoundError) Error() string {
 	return fmt.Sprintf("no descriptor found for reference %q", e.ref.image)
 }
 
+// ImageStructureNotFoundError occurs when the archive file does not exist.
+type ImageStructureNotFoundError struct {
```

Comment:
(Non-blocking: Consider `Archive[File]NotFoundError`.)

Reply:
I was thinking that we might add an equivalent to this for the `oci` directory transport, but I guess that could just be `DirectoryNotFoundError`.

Reply:
I don’t feel strongly about the name … worst case, we can always add type aliases and deprecate old names.

Reply:
It’s just that “structure” seems to be an unexpected word here, where I have to stop and think what it means — and my first guess would be that the archive exists but missing a required subdirectory, or something like that.

Not that I have a better idea if you want keep it generic … `ContainerNotFoundError` is not structurally any better, and “container” would be very misleading in this context.

Reply:
I'm good with `ArchiveFileNotFoundError`